### PR TITLE
fix: docs' links

### PR DESCRIPTION
**Pull request by quasiyoke**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -2,15 +2,15 @@
 //!
 //! [![Latest Version](https://img.shields.io/crates/v/fixnum.svg)](https://crates.io/crates/fixnum)
```
Comment (loyd) [inline src/lib.rs]: Maybe we should replace these badges with ones from README.md. WDYT?

**Review by loyd — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (loyd) [reply src/lib.rs]: Let's remove badges out of docs totally.

File: src/lib.rs [hunk 2]
```diff
@@ -42,11 +42,11 @@
...
-//! | [`cmul`] | `let result: Result<FixedPoint, ArithmeticError> = a.cmul(b)` | Checked multiplication. Returns `Err` on overflow. This is multiplication without rounding, hence it's available only to integer types. |
-//! | [`rmul`] | `let result: Result<FixedPoint, ArithmeticError> = a.rmul(b, RoundMode::Ceil)` | Checked rounded multiplication. Returns `Err` on overflow. Because of provided [`RoundMode`] it's possible across the [`FixedPoint`] values. |
-//! | [`rdiv`] | `let result: Result<FixedPoint, ArithmeticError> = a.rdiv(b, RoundMode::Floor)` | Checked rounded division. Returns `Err` on overflow. Because of provided [`RoundMode`] it's possible across the [`FixedPoint`] values. |
+//! | [`cadd`][cadd] | `let result: Result<FixedPoint, ArithmeticError> = a.cadd(b)` | Checked addition. Returns `Err` on overflow. |
+//! | [`csub`][csub] | `let result: Result<FixedPoint, ArithmeticError> = a.csub(b)` | Checked subtraction. Returns `Err` on overflow. |
+//! | [`cmul`][cmul] | `let result: Result<FixedPoint, ArithmeticError> = a.cmul(b)` | Checked multiplication. Returns `Err` on overflow. This is multiplication without rounding, hence it's available only to integer types. |
```
Comment (loyd) [inline src/lib.rs]: It's also available for `FixedPoint<I>.cmul(I)`

File: src/lib.rs (hunk 1 above)
Comment (quasiyoke) [reply src/lib.rs]: done

File: src/lib.rs (hunk 2 above)
Comment (quasiyoke) [reply src/lib.rs]: described that

**Pull request merged**